packages/render-fidelity-tools/src/components/renderers/babylon-viewer.ts

```diff
@@ -77,7 +77,16 @@ export class BabylonViewer extends LitElement {
     if (this[$scene] != null) {
       this[$scene].dispose();
     }
+
+
     this[$scene] = new Scene(this[$engine]);
+    this[$scene].imageProcessingConfiguration.toneMappingEnabled = true;
+    this[$scene].imageProcessingConfiguration.toneMappingType =
+        ImageProcessingConfiguration.TONEMAPPING_ACES;
+    // tone mapping makes a shift on explosure, so it should be increased be
+    // some factor
+    this[$scene].environmentIntensity = 1 / 0.6;
```

Comment:
I tried some different factor, but the the skybox's intensity is not changed by this value, and the existence of skybox makes it really hard to tell what's the best exposure.  i think it's better to play with the exposure in my disable-skybox pr. Is that ok for you?

Reply:
Interesting; can you ask the babylon folks if there is a different parameter than exposure that does affect the background? Clearly their tone mapping does, and often there is a separate parameter for this. Recall that we will be keeping backgrounds in at least a few of our tests. That's part of why I wanted this done in this PR, so we get the backgrounds right.